<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-S76SA9N" failures="2" tests="3" name="runner.HRISDateCountRunner" time="58.831" errors="0" timestamp="2022-07-06T15:57:44 ICT" skipped="0">
  <testcase classname="runner.HRISDateCountRunner" name="runScenario" time="18.278"/>
  <system-out/>
  <testcase classname="runner.HRISDateCountRunner" name="runScenario" time="21.994">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertTrue(Assert.java:56)
at org.testng.Assert.assertTrue(Assert.java:66)
at definitions.data.HRISBukcetDataDefine.countEquals(HRISBukcetDataDefine.java:251)
at ✽.Count equals 2(file:///D:/Project/Java/TestNG/HRIS/src/test/resources/features/HRISCutiDateCount.feature:129)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase classname="runner.HRISDateCountRunner" name="runScenario" time="18.559">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertTrue(Assert.java:56)
at org.testng.Assert.assertTrue(Assert.java:66)
at definitions.data.HRISBukcetDataDefine.countEquals(HRISBukcetDataDefine.java:251)
at ✽.Count equals 0(file:///D:/Project/Java/TestNG/HRIS/src/test/resources/features/HRISCutiDateCount.feature:129)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
</testsuite> <!-- runner.HRISDateCountRunner -->
